lattice/README.md

```
@@ -48,9 +48,9 @@ introduces the following new concepts:
   examples of
   dopants:
     * VajramDopant: Allows a lattice application to interact with a [
-      `VajramGraph`](../vajram/vajram-krystex/src/main/java/com/flipkart/krystal/vajramexecutor/krystex/VajramGraph.java)
+      `VajramGraph`](../krystex/src/main/java/com/flipkart/krystal/krystex/VajramGraph.java)
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside link text

(MD039, no-space-in-links)